lib/cocoapods/target/build_settings.rb

```diff
@@ -966,6 +999,12 @@ def pod_target_xcconfig_values_by_consumer_by_key
           end
         end
 
+        # @see BuildSettings#should_apply_xctunwrap_fix?
+        def should_apply_xctunwrap_fix?
+          non_library_specs = target.test_specs + target.app_specs
```

Comment:
same question as aggregate targets, what heuristic do we use to apply (or not) the fix?